lib/vr-client.js

```diff
@@ -35,14 +35,16 @@ module.exports = function (hostname, port) {
  * Creates a new run in the given suiteName within the given projectName
  * @param projectName name of the project
  * @param suiteName name of the suite inside the given project
- * @returns {Promise} a promise which resolves a new Run object with the fields run_id, project_id, suite_id.
+ * @param branchName name of the branch the suite was run on
+ * @returns {Promise} a promise which resolves a new Run object with the fields run_id, project_id, suite_id and branch_name.
  * 									  If an error has occurred, the promise will reject with a string containing an error message.
  *
  */
-function createRun (projectName, suiteName) {
+function createRun (projectName, suiteName, branchName) {
```

Comment:
for easy backwards compatibility we could make branchName optional (default it to 'master').